code/modules/medical/surgery_procs.dm

```diff
@@ -1503,21 +1503,24 @@ var/global/list/chestitem_whitelist = list(/obj/item/gnomechompski, /obj/item/gn
 				"<span class='notice'>You begin cauterizing [surgeon == patient ? "your" : "[patient]'s"] wounds closed with [src].</span>",\
 				"<span class='notice'>[patient == surgeon ? "You begin" : "<b>[surgeon]</b> begins"] cauterizing your wounds closed with [src].</span>")
 
-			if (do_mob(patient, surgeon, max((patient.bleeding * 20) - (damage * 2), 0)))
-				surgeon.tri_message(patient, "<span class='notice'><b>[surgeon]</b> cauterizes [patient == surgeon ? "[his_or_her(patient)]" : "[patient]'s"] wounds closed with [src].</span>",\
-					"<span class='notice'>You cauterize [surgeon == patient ? "your" : "[patient]'s"] wounds closed with [src].</span>",\
-					"<span class='notice'>[patient == surgeon ? "You cauterize" : "<b>[surgeon]</b> cauterizes"] your wounds closed with [src].</span>")
-
+			var/dur = max(patient.bleeding * 20 - damage * 2, 0) / 10 SECONDS
```

Comment:
```suggestion
			var/dur = max(patient.bleeding * 20 - damage * 2, 0) DECI SECONDS
```

Reply:
I just moved the / 10 in, to reduce confusion a little